chat-sdk/src/main/java/com/amazon/connect/chat/sdk/repository/ChatService.kt

```diff
@@ -827,7 +841,8 @@ class ChatServiceImpl @Inject constructor(
                     }
                     val oldestInternalTranscriptItemTimeStamp = oldestInternalTranscriptItem.timeStamp
                     val oldestTranscriptItemTimeStamp = oldestTranscriptItem.absoluteTime
-                    if (oldestTranscriptItemTimeStamp <= oldestInternalTranscriptItemTimeStamp) {
+                    if (oldestTranscriptItemTimeStamp.isNotEmpty() && oldestInternalTranscriptItemTimeStamp.isNotEmpty() &&
```

Comment:
Isn't TimeStamp required now?  Or is it optional in Android but required in iOS?  If so, we should try to make it consistent across both platforms.